docker-compose.yaml

```diff
@@ -231,12 +231,14 @@ services:
       TRUSTED_CLIENT_KEYS: "6969"
       ALLOW_DEBUG_AUTH: "True"
       API_ROOT: "http://localhost:8000"
+      DEBUG: "True"
```

Comment:
If I understand correctly, the compose file is only meant for local development, so setting this here shouldn't be a problem?